### ISSUE TITLE
Whether to support java 9?

### ISSUE THREAD

**Issue by mrerhuo** [jersey/jersey#3733]
i creat java9 error list:
![image](https://user-images.githubusercontent.com/9547741/33834584-5edc1c86-debe-11e7-8532-34e3415e0d9f.png)

**Comment by jansupol** [comment]
Have you tried using --add-modules java.xml.bind when starting your application?

**Comment by Bo98** [comment]
@jansupol Aren't Java EE modules marked as deprecated though? I was under the impression that adding `java.xml.bind:jaxb-api` as a dependency was the long-term solution.

From [JEP 261](http://openjdk.java.net/jeps/261):

> Modules that define Java EE APIs, or APIs primarily of interest to Java EE applications, have been deprecated and will be removed in a future release. They are not resolved by default for code on the class path

**Comment by jansupol** [comment]
Yes, the plan is to remove Java EE modules from future versions of JDK. For JDK 9, the modules are still there. As always, JDK not necessarily contains latest versions, so using jaxb dependency you mentioned would be preferable.
